osu.Game.Tests/Visual/Multiplayer/TestSceneLoungeRoomsContainer.cs

```diff
@@ -3,24 +3,26 @@
 
 using System.Linq;
 using NUnit.Framework;
-using osu.Framework.Allocation;
 using osu.Framework.Graphics;
 using osu.Game.Graphics;
 using osu.Game.Online.Rooms;
 using osu.Game.Rulesets.Catch;
 using osu.Game.Rulesets.Osu;
 using osu.Game.Screens.OnlinePlay.Lounge.Components;
+using osu.Game.Tests.Visual.OnlinePlay;
 using osuTK.Graphics;
 using osuTK.Input;
 
 namespace osu.Game.Tests.Visual.Multiplayer
 {
-    public class TestSceneLoungeRoomsContainer : RoomManagerTestScene
+    public class TestSceneLoungeRoomsContainer : OnlinePlayTestScene
     {
+        protected new BasicTestRoomManager RoomManager => (BasicTestRoomManager)base.RoomManager;
+
         private RoomsContainer container;
 
-        [BackgroundDependencyLoader]
-        private void load()
+        [SetUp]
+        public new void Setup() => Schedule(() =>
```

Comment:
I wasn't sure whether this was going to work as expected (by which I mean if this wasn't going to hide the base `Setup()`, but I checked empirically in debug and it looks to be fine. TIL.